docs/source/contributing/guide.md

```diff
@@ -1,11 +1,61 @@
 # Guideline
 
-We welcome contributions to DeepReg, please follow the insutruction for
-[setting up your development environment](setup.html) before editing source code.
-
-- For reporting bugs, or to request features, please
-  [raise an issue](https://github.com/DeepRegNet/DeepReg/issues/new/choose).
-- For fixing bugs, or implementing features, please
-  [send a pull request](pull_request.html).
-- For adding DeepReg Demos, please check the additional requirements in
-  [Add a DeepReg Demo](demo.html).
+We welcome contributions to DeepReg. Please
+[raise an issue](https://github.com/DeepRegNet/DeepReg/issues/new/choose) to report
+bugs, request features, or asking questions. For code contribution, please follow the
+guidelines below.
+
+## Setup
+
+We recommend using conda environment on Linux or Mac machines for code development. The
+setup steps are:
+
+1. [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+2. [Clone or fork](https://github.com/DeepRegNet/DeepReg) the repository.
+3. [Install](../getting_started/install.html) and activate `deepreg` conda environment.
+4. Run `pre-commit install` under the root of this repository `DeepReg/` to install
+   pre-commit hooks.
+
+## Resolve an issue
+
+For resolving an issue, please
+
+1. Create a branch.
+
+   The branch name should start with the issue number, followed by hyphen separated
+   words describing the issue, e.g. `1-update-contribution-guidelines`.
+
+2. Implement the required features or fix the reported bugs.
+
+   There are several guidelines for commit, coding, testing, and documentation.
+
+   a. Please create commits with meaningful commit messages.
+
+   The commit message should start with `Issue #<issue number>:`, for instance
+   `Issue #1: add commit requirements.`
+
+   b. Please write or update unit-tests for the added or changed functionalities.
+
+   Pull request will not be approved if test coverage is decreased. Check
+   [testing guidelines](test.html) for further details.
+
+   c. Please write meaningful docstring and documentations for added or changed code.
+
+   We use
+   [Sphinx docstring format](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html).
+
+   d. Please update the
+   [CHANGELOG](https://github.com/DeepRegNet/DeepReg/blob/main/CHANGELOG.md) regarding
+   the changes.
```

Comment:
The points under a/b/c don't display all that nicely (or obviously) as subpoints in the RTD. Is it possible to change that?

See attached photo. Can we indent the bits marked in red?

![image](https://user-images.githubusercontent.com/7175543/100349173-f3d58000-2fdf-11eb-9229-62398a6de4d3.png)

Reply:
ooops will add more space ;)

Reply:
solved :)